src/ElementInfoUtils.h

```diff
@@ -0,0 +1,74 @@
+//
+//  MergeElementInfo.h
+//  drafter
+//
+//  Created by Jiri Kratochvil on  20/02/2018
+//  Copyright (c) 2018 Apiary Inc. All rights reserved.
+//
+
+#include "ElementData.h"
+
+#ifndef DRAFTER_MERGEELEMENTINFO_H
+#define DRAFTER_MERGEELEMENTINFO_H
+
+namespace drafter
+{
+
+    template <typename T, bool IsPrimitive = is_primitive<T>()>
+    struct Merge;
+
+    template <typename T>
+    struct Merge<T, true> {
+        using Info = ElementInfo<T>;
+        using Container = std::deque<Info>;
+
+        ElementInfo<T> operator()(Container container) const
```

Comment:
You have defined `Info` above. Why not use here and below?

Reply:
It is header, I want to avoid to dirty global space. Local naming is better in this case